src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -31,20 +31,18 @@ namespace Microsoft.PowerShell.Commands
 {
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-windows`
Certificate Provider tests.Error occurred in Describe block
```powershell
Expected 2, but got 3.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp5.0\win7-x64\publish\Modules\Pester\4.8.0\Functions\Assertions\Should.ps1:186 char:9
```
CmsMessage cmdlets thorough tests.Error occurred in Describe block
```powershell
Expected 2, but got 3.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp5.0\win7-x64\publish\Modules\Pester\4.8.0\Functions\Assertions\Should.ps1:186 char:9
```